pkgs/by-name/tr/trelby/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "trelby";
+  version = "2.4.15";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "trelby";
+    repo = "trelby";
+    tag = "${version}";
+    hash = "sha256-CTasd+YlRHjYUVepZf2RDOuw1p0OdQfJENZamSmXXFw=";
+  };
+
+  build-system = [
+    python3Packages.setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    wxpython
+    lxml
+    reportlab
+  ];
+
```

Comment:
Is there a `--version` flag for this program? If so, can you add `versionCheckHook` to `nativeCheckInputs`?

Reply:
Ok!